### ISSUE TITLE
Integrate with public filters

### ISSUE THREAD

**Issue by arranbartish** [janotav/ali-idea-plugin#54]
# Reasoning

A team have a public filter defined on the server that helps refine what is the most pressing/valuable defects that need to be worked on. Currently it seems that each dev must replicated these filters with the plugin to get the same list. 

# Questions 

- Are the public/private filters of ALM/ALI exposed via the rest interface? 
- Could we get the definition of the filter and so update the plugin to match? (replicating the result)
- Could is a GET gor a public private filter exposed by ALM/ALI where we can just output the result?

Happy to contribute a pull request with some advice/info about what I can do via ALM/ALI's exposed interfaces.

**Comment by arranbartish** [comment]
Is this project still alive?

**Comment by janotav** [comment]
Nope, this project is no longer actively maintained. I have lost any contact with the ALM/AGM group inside HPE and I'm no longer able to do any reasonable development.

I should have put a sign about it - my apologies for not doing that, I'll fix this asap.
